src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const COST_PER_DAY = 40;
+  const SHORT_TERM = 3;
+  const LONG_TERM = 7;
+
+  let discount = 0;
+
+  if (days >= SHORT_TERM) {
+    discount = 20;
+  }
+
+  if (days >= LONG_TERM) {
+    discount = 50;
+  }
+
+  return (COST_PER_DAY * days) - discount;
```

Comment:
fix the checklist - point 5
![image](https://github.com/mate-academy/js_task-transportation-on-vacation/assets/22824431/01ee177a-f99b-43ff-8f09-5b3cf223df8f)